docs/get-started/nextjs.md

```diff
@@ -885,54 +989,83 @@ You can refer to the [Install `sharp` to Use Built-In Image Optimization](https:
 
 ## API
 
-### Parameters
+### Modules
 
-This framework contributes the following [parameters](../writing-stories/parameters.md) to Storybook, under the `nextjs` namespace:
+The `@storybook/nextjs` package exports a number of modules that enables you to [mock](#mocking-modules) Next.js's internal behavior.
 
-#### `appDirectory`
+#### `@storybook/nextjs/export-mocks`
 
-Type: `boolean`
+Type: `{ getPackageAliases: ({ useESM?: boolean }) => void }`
 
-Default: `false`
+`getPackageAliases` is a helper to generate the aliases needed to set up [portable stories](#portable-stories).
 
-If your story imports components that use `next/navigation`, you need to set the parameter `nextjs.appDirectory` to `true`. Because this is a parameter, you can apply it to a [single story](../api/parameters.md#story-parameters), [all stories for a component](../api/parameters.md#meta-parameters), or [every story in your Storybook](../api/parameters.md#project-parameters). See [Next.js Navigation](#nextjs-navigation) for more details.
+```ts
+import type { Config } from 'jest';
+import nextJest from 'next/jest.js';
+// 👇 import the utility function
+import { getPackageAliases } from '@storybook/nextjs/export-mocks';
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+});
+
+// Add any custom config to be passed to Jest
+const config: Config = {
+  coverageProvider: 'v8',
+  testEnvironment: 'jsdom',
+  // Add more setup options before each test is run
+  // setupFilesAfterEnv: ['<rootDir>/jest.setup.ts'],
+  moduleNameMapper: {
+    ...getPackageAliases(), // 👈 add the utility as a moduleNameMapper
+  },
+};
 
-#### `navigation`
+// createJestConfig is exported this way to ensure that next/jest can load the Next.js config which is async
+export default createJestConfig(config);
+```
```

Comment:
This is just [the example from the Next.js docs](https://nextjs.org/docs/app/building-your-application/testing/jest#manual-setup) plus the utility function. So it can be condensed a lot of we want to.